spec/lib/rex/proto/postgresql/client_spec.rb

```diff
@@ -20,4 +20,47 @@
   end
 
   it_behaves_like 'session compatible SQL client'
+
```

Comment:
As we talked about on call previously - could we test the `detect_platform_and_arch` function, and mock out the results of `query('select version()')` to examples of real versions that you've queried :+1:

That way if future folk change the regex implementation and break things, we'll have tests covering this